core/source/api/opencmiss/zinc/fieldfiniteelement.h

```diff
@@ -481,23 +481,36 @@ ZINC_API int cmzn_field_stored_string_destroy(
  * Creates a field which returns 1 on 2-D faces and 1-D lines considered as
  * exterior to their top-level element, and 0 elsewhere.
  *
- * @param field_module  Region field module which will own new field.
+ * @param fieldmodule  Region field module which will own new field.
  * @return  Handle to new field, or NULL/invalid handle on failure.
  */
 ZINC_API cmzn_field_id cmzn_fieldmodule_create_field_is_exterior(
-	struct cmzn_fieldmodule *field_module);
+	cmzn_fieldmodule_id fieldmodule);
 
 /**
  * Creates a field which returns 1 on 2-D faces and 1-D lines considered
  * to lie on a specified face of their top-level element, and 0 elsewhere.
  *
- * @param field_module  Region field module which will own new field.
+ * @param fieldmodule  Region field module which will own new field.
  * @param face  The enumerated face type, defined with respect to the
  * top-level element.
  * @return  Handle to new field, or NULL/invalid handle on failure.
  */
 ZINC_API cmzn_field_id cmzn_fieldmodule_create_field_is_on_face(
-	struct cmzn_fieldmodule *field_module, cmzn_element_face_type face);
+	cmzn_fieldmodule_id fieldmodule, cmzn_element_face_type face);
+
+/**
+ * Creates a field whose value equals source field calculated at the lookup node
+ * instead of the domain location requested.
+ *
+ * @param fieldmodule  Region field module which will own new field.
+ * @param source_field  Field to evaluate.
```

Comment:
@param doesn't match true argument name: source_field vs. field. If you copied it from somewhere fix that as well!